src/share-links.tsx

```diff
@@ -9,7 +9,7 @@ import SampleIndicatorCard from "./components/SampleIndicatorCard";
  */
 export const LINKABLE_COMPONENTS: { [key: string]: React.ComponentType<any> } =
   {
-    sample: SampleIndicatorCard,
+    sample: SamplePaper,
```

Comment:
At the end of the day it won't be the the full page, but only the components on the page that are Linkable.